ipn/ipnstate/ipnstate.go

```diff
@@ -64,6 +63,26 @@ type Status struct {
 	User map[tailcfg.UserID]tailcfg.UserProfile
 }
 
+// TailnetStatus is a subset of status information related to the
+// current tailnet.
```

Comment:
I think this field itself can describe any tailnet, like when we have multiple at once.

I'd drop this "current" and move "current" to the currently-undocumented Tailnet field above.

Maybe call that field CurrentTailnet?